roles/bootstrap/tasks/main.yml

```diff
@@ -108,18 +154,39 @@
     remote_src: yes
     mode: u+rwx,g+rx,o+rx
 
-- name: Mount host '/proc' folder
-  ansible.posix.mount:
-    path: "{{ image_mount_point }}/proc"
-    src: /proc
-    opts: bind
-    state: mounted
-    fstype: none
+- name: "Create the '{{ cluster_user }}' user"
+  user:
+    name: "{{ cluster_user }}"
+    comment: Cluster User with root access
+    uid: 1041
+    group: root
 
-- name: Mount host '/dev' folder
-  ansible.posix.mount:
-    path: "{{ image_mount_point }}/dev"
-    src: /dev
-    opts: bind
-    state: mounted
-    fstype: none
+- name: "Create the '{{ cluster_user }}' user in chroot filesystem"
+  shell: chroot {{ image_mount_point }} useradd -u 1041 -m {{ cluster_user }}
+  become: yes
+
+- name: "Create home folder for {{ cluster_user }}"
```

Comment:
Single quote around the cluster_user var so it looks nicer on the log:
"Create home folder for '{{ cluster_user }}'"